PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -45,6 +71,10 @@
      ),
 )
 
+(run2_miniAOD_80XLegacy | run2_nanoAOD_92X | run2_nanoAOD_94X2016 | run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94XMiniAODv2\
```

Comment:
This and the other cases below should just be `run2_nanoAOD_106Xv2 |  run3_nanoAOD_122 `